### PR TITLE
Login/signup

### PR DESCRIPTION
**Pull request by MuskanChhatrasal**
Login/Signup page done

### CODE REVIEW

**Comment by MuskanChhatrasal** [comment]
> Login/Signup page done

Live Preview: 
https://deploy-preview-5--eager-williams-1c3e97.netlify.app/pages/login

**Review by aayushjain-code — APPROVED** [review]
Very nice UI looks good from my side ...
try to align the login & signup form to bit more center

**Comment by Hamza-777** [comment]
The UI looks really nice. The text color inside your inputs is set to white so change that.

**Review by vishal804 — APPROVED** [review]

File: components/Login.css [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+}
+
+form .input-box input {
+  height: 100%;
+  width: 100%;
+  outline: none;
+  padding: 0 1rem;
+  font-size: 1.6rem;
+  font-weight: 400;
+  color: var(--text-white);
```
Comment (Hamza-777) [inline components/Login.css]: Here you set the text color to white inside the input which matches your background. So, change that.

**Comment by vishal804** [comment]
Everything Look Good Nice UI

**Review by Hamza-777 — APPROVED** [review]
Everything looks good just make the changes and you are good to go ahead and merge it.

**Comment by MuskanChhatrasal** [comment]
Done!!

**Comment by MuskanChhatrasal** [comment]
Done!! Thankyou

**Comment by MuskanChhatrasal** [comment]
Thankyou!!

**Pull request merged**